internal/features/five_point_oh.go

```diff
@@ -31,19 +26,3 @@ func DeprecatedInFivePointOh(deprecationMessage string) string {
 func FivePointOh() bool {
 	return false
 }
-
-// FivePointOhBeta returns whether this provider is running in 5.0 mode
-// which is an opt-in Beta of the changes coming in 5.0.
-//
-// This exists to allow breaking changes to be piped through the provider
-// during the development of 4.x until 5.0 is ready.
-//
-// The environment variable `ARM_FIVEPOINTZERO_BETA` has been added
-// to facilitate testing. But it should be noted that
-// `ARM_FIVEPOINTZERO_BETA` is ** NOT READY FOR PUBLIC USE ** and
-// ** SHOULD NOT BE SET IN PRODUCTION ENVIRONMENTS **
-// Setting `ARM_FIVEPOINTZERO_BETA` will cause irreversible changes
-// to your state.
-func FivePointOhBeta() bool {
-	return FivePointOh() || strings.EqualFold(os.Getenv("ARM_FIVEPOINTZERO_BETA"), "true")
-}
```

Comment:
We need to keep this in so that changes using the 5.0 flag can be tested.

I would rename this function to `FivePointOh()` and remove the once defined on lines 31-33